src/main/java/jp/co/future/uroborosql/SqlAgentImpl.java

```diff
@@ -34,50 +40,156 @@
+import jp.co.future.uroborosql.event.TransformSqlEvent;
 import jp.co.future.uroborosql.exception.EntitySqlRuntimeException;
 import jp.co.future.uroborosql.exception.OptimisticLockException;
 import jp.co.future.uroborosql.exception.PessimisticLockException;
 import jp.co.future.uroborosql.exception.UroborosqlRuntimeException;
 import jp.co.future.uroborosql.exception.UroborosqlSQLException;
+import jp.co.future.uroborosql.fluent.Procedure;
+import jp.co.future.uroborosql.fluent.SqlBatch;
 import jp.co.future.uroborosql.fluent.SqlEntityDelete;
 import jp.co.future.uroborosql.fluent.SqlEntityQuery;
 import jp.co.future.uroborosql.fluent.SqlEntityUpdate;
+import jp.co.future.uroborosql.fluent.SqlQuery;
+import jp.co.future.uroborosql.fluent.SqlUpdate;
 import jp.co.future.uroborosql.mapping.EntityHandler;
 import jp.co.future.uroborosql.mapping.MappingColumn;
 import jp.co.future.uroborosql.mapping.MappingUtils;
 import jp.co.future.uroborosql.mapping.TableMetadata;
+import jp.co.future.uroborosql.parameter.Parameter;
+import jp.co.future.uroborosql.parser.SqlParserImpl;
+import jp.co.future.uroborosql.store.SqlResourceManager;
+import jp.co.future.uroborosql.tx.LocalTransactionManager;
+import jp.co.future.uroborosql.tx.TransactionManager;
 import jp.co.future.uroborosql.utils.CaseFormat;
+import jp.co.future.uroborosql.utils.ObjectUtils;
 
 /**
  * SQL実行用クラス。
  *
  * @author H.Sugimoto
  */
-public class SqlAgentImpl extends AbstractAgent {
+public class SqlAgentImpl implements SqlAgent {
 	/** ロガー */
 	private static final Logger LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.log");
 
 	/** SQLロガー */
 	private static final Logger SQL_LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.sql");
 
+	/** SQLカバレッジ用ロガー */
+	private static final Logger COVERAGE_LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.sql.coverage");
+
 	/** パフォーマンスロガー */
 	private static final Logger PERFORMANCE_LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.performance");
 
+	/** REPLロガー */
+	private static final Logger REPL_LOG = LoggerFactory.getLogger("jp.co.future.uroborosql.repl");
+
+	/** ログ出力を抑止するためのMDCキー */
+	private static final String SUPPRESS_PARAMETER_LOG_OUTPUT = "SuppressParameterLogOutput";
+
+	/** ExecutionContext属性キー:リトライカウント */
+	private static final String CTX_ATTR_KEY_RETRY_COUNT = "__retryCount";
+
+	/** ExecutionContext属性キー:バインドパラメータコメントの出力有無 */
+	private static final String CTX_ATTR_KEY_OUTPUT_BIND_COMMENT = "__outputBindComment";
+
+	/** 例外発生にロールバックが必要なDBでリトライを実現するために設定するSavepointの名前 */
+	private static final String RETRY_SAVEPOINT_NAME = "__retry_savepoint";
+
+	/** IN句に渡すパラメータのMAXサイズ */
+	private static final int IN_CLAUSE_MAX_PARAM_SIZE = 1000;
+
 	/** 経過時間のフォーマッタ */
 	private static final DateTimeFormatter ELAPSED_TIME_FORMAT = DateTimeFormatter.ofPattern("HH:mm:ss.SSSSSS");
 
+	/** カバレッジハンドラ */
+	private static final AtomicReference<CoverageHandler> coverageHandlerRef = new AtomicReference<>();
```

Comment:
Unless there is a special reason, I think it is better to use an upper snake case.

```suggestion
	private static final AtomicReference<CoverageHandler> COVERAGE_HANDLER_REF = new AtomicReference<>();
```

Reply:
UPPER SNAKE CASE is correct and has been corrected.